test/format/config.c

```diff
@@ -962,6 +962,12 @@ config_transaction(void)
     if (g.c_logging && config_is_perm("logging")) {
         if (g.c_prepare)
             config_single("ops.prepare=off", false);
+        /* Rollback-to-stable doesn't operate on logged tables. */
+        if (g.c_txn_rollback_to_stable && config_is_perm("transaction.rollback_to_stable"))
+            testutil_die(
+              EINVAL, "rollback to stable and logging can not be configured at the same time");
+        else if (g.c_txn_rollback_to_stable)
+            config_single("transaction.rollback_to_stable=off", false);
```

Comment:
I don't understand the reason for this change -- why does it improve anything?

Reply:
I have replied [here](https://github.com/wiredtiger/wiredtiger/pull/6379#issuecomment-799034913)